tools/deployment/make_linux_package.sh

```diff
@@ -117,6 +117,7 @@ function main() {
   platform OS
   distro $OS DISTRO
 
+  touch "$BUILD_DIR/$PACKAGE_NAME$(get_pkg_suffix)"
```

Comment:
Let me try that out. Apologies for the delay, but I'm now running into a problem with the make deps script now that is preventing me from verifying that will work (homebrew can no longer as root, but i haven't nailed the set of permissions necessary to get everything to install correctly). BTW is that a known issue? Everything was working fine 3 days ago :)